packages/core/src/payment/payment-strategy-registry.ts

```diff
@@ -51,6 +51,16 @@ export default class PaymentStrategyRegistry extends Registry<
     }
 
     private _getToken(paymentMethod: PaymentMethod): PaymentStrategyType {
+        const features = this._store.getState().config.getStoreConfig()?.checkoutSettings.features;
+
+        if (
+            paymentMethod.id === 'squarev2' &&
+            features &&
+            features['PROJECT-4113.squarev2_web_payments_sdk']
+        ) {
+            throw new Error('SquareV2 requires using registryV2');
```

Comment:
We'll have two `squarev2` payment methods at the same time, one is the old one, the other is a monorepo package.
They share exactly the same method id, so I made this PR to skip the old one when the experiment is on.

Reply:
Ok so the old one isn't removed from V1 registry yet?

Reply:
Yes, they want to put the new one behind the experiment, `PROJECT-4113.squarev2_web_payments_sdk`.

Reply:
Ok, preferably we want a way so that integration team can toggle between old vs new implementations without making a change to the `core` package. But I can't think of a quick way to do it. Something to think about in the future.

Reply:
Sure, thanks DC.